sdk/ml/azure-ai-ml/azure/ai/ml/entities/_manage_clouds.py

```diff
@@ -0,0 +1,7 @@
+from azure.ai.ml._azure_environments import (_add_cloud, _update_cloud)
```

Comment:
We should not need this file if we are not exposing it as a functionality. We can keep this file for internal use if needed.